.ci/update-github-page.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable "GITHUB_TOKEN"
+checkForVariable "READ_ONLY_TOKEN"
+
+checkout_from https://github.com/checkstyle/contribution
+
+cd .ci-temp/contribution/releasenotes-builder
+mvn -e --no-transfer-progress clean compile package
+cd ../../../
+
+if [ -d .ci-temp/checkstyle ]; then
+  cd .ci-temp/checkstyle/
+  git reset --hard origin/master
+  git pull origin master
+  git fetch --tags
+  cd ../../
+else
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/checkstyle
+  cd ../
+fi
+
+cd .ci-temp/checkstyle
+
+curl \
+ https://api.github.com/repos/checkstyle/checkstyle/releases \
+ -H "Authorization: token $GITHUB_TOKEN" \
+ -o /var/tmp/cs-releases.json
+
+TARGET_RELEASE_NUM=$1
```

Comment:
put this line above checkForVariable

script need to fail if parameter is not defined please put validation on parameter

Reply:
Changes Done

Reply:
Why you keep it here ?

Reply:
Changes Done. Did that in hurry.